scalableminds/webknossos
# Switch default volume layer source for advanced annotation creation modal

When creating a new annotation using the more advanced/detailed modal from the overflow menu, the default option for new annotations is to create an empty volume layer. Our internal discussion showed, that more often than not, you would likely want to base your volume annotation on an existing segmentation layer instead.

This PR adds the following changes:
- switch the default volume layer to existing segmentation. layer instead of empty one
- a small cosmetic CSS tweak to prevent longer mag labels from wrapping, e.g. `1024-1024-512`

switch default volume layer to existing segmentation layers instead of empty volume layer when using detailed modal to create a new annotation

<img width="501" alt="image" src="https://user-images.githubusercontent.com/1105056/195082582-e8eee2f0-16a7-4ac6-beb7-2717fc9da3df.png">


### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Go to dashboard
- Create new annotation using the overflow menu (three dots ...)
- Any existing segmentation layer should be pre-selected by default

### Issues:
- fixes None

------
(Please delete unneeded items, merge only when none are left open)
